dev/integration_tests/flutter_gallery/lib/demo/cupertino/cupertino_navigation_demo.dart

```diff
@@ -763,8 +763,8 @@ class CupertinoDemoTab3 extends StatelessWidget {
               decoration: BoxDecoration(
                 color: CupertinoTheme.of(context).scaffoldBackgroundColor,
                 border: const Border(
-                  top: BorderSide(color: Color(0xFFBCBBC1), width: 0.0),
-                  bottom: BorderSide(color: Color(0xFFBCBBC1), width: 0.0),
+                  top: BorderSide(color: Color(0x80BCBBC1), width: 2.0, strokeAlign: BorderSide.strokeAlignCenter),
```

Comment:
While it might look the same in some cases, I don't think this is exactly the same thing, because I think it depends on the anti-aliasing algorithm used for drawing the stroke, and the device pixel ratio.

Reply:
I can make it affect only with borderRadius for now then :(